modules/autofill/autofill-api/build.gradle

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+    id 'kotlin-android-extensions'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
```

Comment:
I've seen some modules using 
apply from: "$rootProject.projectDir/spotless.gradle" instead of this. I am not sure what should it be.

Reply:
It depends on what you need, is autofill-api an Android module? If yes then use android-library, else I don't usually use anything because the api module is a fairly straightforward interface(s). 
Maybe we should add a kotlin-library gradle file for pure kotlin libraries because now that I look at it I don't think we are running spotless in the non-Android api modules which is probably fine but not ideal.